### PR TITLE
docs(zh-CN): customize branding

### PR DESCRIPTION
**Pull request by gao-sun**
配置品牌

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/customize-sie/configure-branding.mdx [hunk 1]
```diff
@@ -1 +1,22 @@
...
+import DarkModeNote from './fragments/_dark-mode-note.md';
+import SaveChanges from './fragments/_save-changes.md';
+
+# 配置品牌
+
+<SaveChanges />
+
+## 深色模式
+
+_开启深色模式_ 决定了当 Logto 检测到相关系统偏好时，是否让 UI 有一种「深色的感觉」。
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/customize-sie/configure-branding.mdx:10]: nit: should align with `启用` as below?

开启 → 启用

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/customize-sie/configure-branding.mdx [hunk 2]
```diff
@@ -1 +1,22 @@
...
+
+## 品牌色
+
+_品牌色_ 将影响组建的主要颜色，包括但不限于按钮、链接等。试试将 Logto 蓝紫色替换成你的品牌色。
```
Comment (darcyYe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/customize-sie/configure-branding.mdx:14]: `组建` -> `组件`?

**Review by darcyYe — APPROVED** [review]
LGTM

**Pull request merged**